### ISSUE TITLE
HTTP error while running migrado run

### ISSUE THREAD

**Issue by Nargonath** [protojour/migrado#11]
As I couldn't use the official image initially, I ended up writing my own Docker image which installs `migrado` through `pip`. I was able to create the initial migration but I have an error now when I try to run it:

```shell
docker run \
		--mount type=bind,source=./,target=/app \
		-e MIGRADO_DB \
		-e MIGRADO_HOST \
		-e MIGRADO_USER \
		-e MIGRADO_PASS \
		--rm \
		arango-migrations run
```

I get:

```
docker run --mount type=bind,source=./,target=/app -e MIGRADO_DB -e MIGRADO_HOST -e MIGRADO_USER -e MIGRADO_PASS --rm arango-migrations run
Traceback (most recent call last):
  File "/usr/local/bin/migrado", line 8, in <module>
    sys.exit(migrado())
             ^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/migrado/migrado.py", line 475, in run
    db_client = MigrationClient(tls, host, port, username, password, db, state_coll, timeout)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/migrado/db_client.py", line 53, in __init__
    self.db_client = ArangoClient(f'{self.protocol}://{host}:{port}', http_client=http_client)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/arango/client.py", line 119, in __init__
    self._sessions = [self._http.create_session(h) for h in self._hosts]
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/arango/client.py", line 119, in <listcomp>
    self._sessions = [self._http.create_session(h) for h in self._hosts]
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/arango/http.py", line 175, in create_session
    total=self._retry_attempts,
          ^^^^^^^^^^^^^^^^^^^^
AttributeError: 'HTTPClient' object has no attribute '_retry_attempts'
make: *** [Makefile:13: run] Error 1
```

I tried using `python:3.11-bookworm` and `python:3.12-bookworm` but makes no difference.

**Comment by tangram** [comment]
This was due to a breaking change in the `python-arango` HTTP client from a while back. I've updated Migrado to require the latest version, and use the default HTTP client as it now supports timeout.

Fixed in `0.6.9`, which is now in PyPI and the latest Docker image.

**Comment by Nargonath** [comment]
Cool thank you.
